libs/dynsigner/src/lib.rs

```diff
@@ -31,7 +31,6 @@ impl Signer for DynSigner {
 }
 
 /// Keypair or Pubkey depending, could be one of that based on parameters of the CLI command.
-/// For --print-only we want to permit to pass only pubkey not the keypair in real.
```

Comment:
Don't you want to leave here some info that this is useful for `print` when only pubkey could be relevant?